src/librbd/io/ImageRequest.cc

```diff
@@ -521,19 +521,9 @@ template <typename I>
 uint64_t ImageWriteRequest<I>::append_journal_event() {
   I &image_ctx = this->m_image_ctx;
```

Comment:
nit: do we need this helper at all now? May be just call `image_ctx.journal->append_write_event` directly, as you do for other request?

Reply:
Yeah, this is actually a bit more than a helper - it's abstract so that `AbstractImageWriteRequest<I>::send_request()` can call it generically for all request types.

Reply:
Ah, ok, never mind